commands/src/main/java/org/wildfly/extras/creaper/commands/auditlog/AddFileHandler.java

```diff
@@ -0,0 +1,133 @@
+package org.wildfly.extras.creaper.commands.auditlog;
+
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.GroovyXmlTransform;
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.Subtree;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.offline.OfflineCommand;
+import org.wildfly.extras.creaper.core.offline.OfflineCommandContext;
+import org.wildfly.extras.creaper.core.online.OnlineCommand;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.OperationException;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
+
+public class AddFileHandler implements OnlineCommand, OfflineCommand {
```

Comment:
This name is too generic. E.g. in `org.wildfly.extras.creaper.commands.logging`, we have `AddPeriodicRotatingFileLogHandler`. This one should be called something like `AddAuditLogFileHandler`.